index.html

```diff
@@ -1,10 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
+<<<<<<< Updated upstream
 <head>
     <meta charset="UTF-8">
     <title>Title</title>
 </head>
 <body>
 
 </body>
+=======
+    <head>
+        <title>Frontend workshops</title>
+        <link rel="stylesheet" href="./css/main.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <BODY>
```

Comment:
```suggestion
    <body>
```
Ze względu na konwencję.